.circleci/config.yml

```
@@ -107,3 +108,19 @@ jobs:
           name: Run system tests
           command: |
             COVERAGE=false bundle exec rspec spec/system/
+
+
+workflows:
+  test-build-and-publish-image:
+    jobs:
+      - run-tests
+      - wait-for-approval:
+          type: approval
+      - docker/publish:
+          cache_from: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME:latest
+          image: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+          tag: latest
```

Should this also have a commit tag so it's easy to reference a particular version? Not sure if that happens automatically.

```suggestion
          tag: latest,$CIRCLE_SHA1
```

---

Yep, we can do that. I couldn't figure out yesterday how to get the SHA, hehe.